src/test/java/org/hobbit/core/components/BenchmarkControllerTest.java

```diff
@@ -210,118 +215,125 @@ public DummyPlatformController(String sessionId) {
-                        // amount of
-                        // time and terminate after that
-                        DummyComponentExecutor taskGenExecutor = new DummyComponentExecutor(
-                                new AbstractTaskGenerator() {
-                                    @Override
-                                    public void run() throws Exception {
-                                        LOGGER.debug("Task Generator started...");
-                                        super.run();
-                                    }
-
-                                    @Override
-                                    protected void generateTask(byte[] data) throws Exception {
-                                    }
-                                }) {
-                            @Override
-                            public void run() {
-                                super.run();
-                                try {
-                                    sendToCmdQueue(Constants.HOBBIT_SESSION_ID_FOR_BROADCASTS,
-                                            Commands.DOCKER_CONTAINER_TERMINATED,
-                                            RabbitMQUtils.writeByteArrays(null,
-                                                    new byte[][] { RabbitMQUtils.writeString(containerId) },
-                                                    new byte[] { (byte) 0 }),
-                                            null);
-                                } catch (IOException e) {
-                                    e.printStackTrace();
-                                    success = false;
-                                }
-                            }
-                        };
-                        taskGenExecutors.add(taskGenExecutor);
-                        Thread t = new Thread(taskGenExecutor);
-                        taskGenThreads.add(t);
-                        t.start();
-
-                        cmdChannel.basicPublish("", replyTo, replyProps,
-                                RabbitMQUtils.writeString(containerId));
-                    } else if (startCommandJson.contains(EVAL_IMAGE)) {
-                        cmdChannel.basicPublish("", replyTo, replyProps,
-                                RabbitMQUtils.writeString(containerId));
-                        sendToCmdQueue(this.sessionId, Commands.EVAL_STORAGE_READY_SIGNAL, null, null);
-                    } else {
-                        LOGGER.error("Got unknown start command. Ignoring it.");
-                    }
-                } catch (IOException e) {
-                    LOGGER.error("Exception while trying to respond to a container creation command.", e);
-                }
-            }
+        	createDummyComponent(command, data, sessionId, props);
         }
+        
+        public void createDummyComponent(byte command, byte[] data, String sessionId, AMQP.BasicProperties props) {
```

Comment:
The method name is not really helpful. Please try to find a name that states what the method does.